built_value_generator/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: built_value_generator
```

Comment:
Please run tool/use_local_deps and update the PR; it will update all the yaml files to use the unreleased version, so the presubmit results are meaningful.

Reply:
ran the tool and regenerated the example files.